rotkehlchen/accounting/structures/types.py

```diff
@@ -37,8 +37,8 @@
 class ActionType(DBCharEnumMixIn):
     TRADE = 1
     ASSET_MOVEMENT = 2
-    EVM_TRANSACTION = 3
-    LEDGER_ACTION = 4
```

Comment:
Should we delete ledger action from the db too?

Reply:
The PR is doing that

Reply:
yes, from the schema is what I mean

Reply:
done